app/ts/simulation/services/EthereumClientService.ts

```diff
@@ -321,7 +331,7 @@ export class EthereumClientService {
 						loggersAddress: log.address,
 						data: 'data' in log && log.data !== undefined ? log.data : new Uint8Array(),
 						topics: 'topics' in log && log.topics !== undefined ? log.topics : [],
-					})),
+					})).filter((x) => x.loggersAddress !== 0x0n), //TODO, keep eth logs
```

Comment:
Create issue for TODO or address it in this PR.

Reply:
created issue about it